test/e2e/unifiedpush_test.go

```diff
@@ -0,0 +1,51 @@
+package e2e
+
+import (
+	"testing"
+	"time"
+
+	apis "github.com/aerogear/unifiedpush-operator/pkg/apis"
+	aerogearv1alpha1 "github.com/aerogear/unifiedpush-operator/pkg/apis/aerogear/v1alpha1"
+	framework "github.com/operator-framework/operator-sdk/pkg/test"
+	"github.com/operator-framework/operator-sdk/pkg/test/e2eutil"
+)
+
+var (
+	retryInterval        = time.Second * 5
+	timeout              = time.Second * 200
+	cleanupRetryInterval = time.Second * 1
+	cleanupTimeout       = time.Second * 5
+)
+
+func TestUnifiedpush(t *testing.T) {
+	unifiedpushList := &aerogearv1alpha1.UnifiedPushServerList{}
```

Comment:
I know there are no real tests in this file, but I have the feeling we should have the meta passed as here: https://github.com/operator-framework/operator-sdk/blob/master/doc/test-framework/writing-e2e-tests.md#2-register-types-with-framework-scheme

```
unifiedpushList := &aerogearv1alpha1.UnifiedPushServerList{
		TypeMeta: metav1.TypeMeta{
			Kind:       "UnifiedPushServer",
			APIVersion: "aerogear.org/v1alpha1",
		},
	}
```

Reply:
I removed the `TypeMeta` section from original commit as @grdryn mentioned it could make a confusion when the versions of apiversion will change. See [this comment](https://github.com/aerogear/unifiedpush-operator/pull/9#discussion_r284798859).

Looking now at the other test implementations ([keycloak](https://github.com/integr8ly/keycloak-operator/blob/master/test/e2e/deployment_test.go#L29), [gitea](https://github.com/integr8ly/gitea-operator/blob/master/test/e2e/gitea_test.go#L27), [memcached](https://github.com/operator-framework/operator-sdk-samples/blob/master/memcached-operator/test/e2e/memcached_test.go#L41)), and seeing that `TypeMeta` is always used. Any thoughts on final resolution of this? :)

Reply:
leaving it to @grdryn then :)

but recommendation would be doing what memcached operator does